src/pages/shinung/Login/Login.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react/cjs/react.development';
+import './Login.scss';
+
+const LoginShinung = () => {
+  const navigate = useNavigate();
+  const [btn, setBtn] = useState(false);
+
+  const goToMain = () => {
+    navigate('/main-shinung');
+  };
+
+  const [inputValues, setInputValues] = useState({
+    email: '',
+    password: '',
+  });
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setInputValues({
+      ...inputValues,
+      [name]: value,
+    });
+    handleBtn();
+  };
+
+  const handleBtn = () => {
+    const regEmail =
+      /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/;
```

Comment:
Codacy has a fix for the issue: [Unnecessary escape character: \..](https://app.codacy.com/gh/wecode-bootcamp-korea/29-React-Westagram-3/pullRequest?prid=10028328)
```suggestion
      /^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/;
```